vms/platformvm/utxo/owner_id.go

```diff
@@ -0,0 +1,30 @@
+package utxo
```

Comment:
license header, pls. rename camino_owner_id.
beside this maybe look what the addressable interface provides.

Reply:
addressable returns [][]byte without any info about threshold

Reply:
ava also use exactly this approach to ownerIDs in verifySpend